stories/Atom/Global/Spacing/spacing.stories.mdx

```diff
@@ -0,0 +1,42 @@
+import { Meta, Story } from '@storybook/addon-docs';
+import { Spacing } from './spacing';
+
+<Meta title="Atoms/Spacing" component={Spacing} />
+
+<style>{`
+  div#story--atoms-spacing--spacing {
+    box-shadow: 0 1px 8px rgb(0 0 0 / 15%);
+    box-sizing: inherit;
+  }
+  .subheading {
+    font-weight: 700;
+    font-size: 18px;
+    color: #666;
+    letter-spacing: 1.5px;
+    line-height: 24px;
+    margin-bottom: 12px;
+    margin-top: 40px;
+  }  
+`}</style>
+
+# Spacing Component
+Space units create visual rhythm and balance.
+
+<ul>
+  <li>Margin is a css property for spacing outside of a component.</li>
+  <li>Padding is a css property for spacing inside of a component.</li>
+  <li>We can use <code>spacing</code> in any component to sets the margin and padding area on all four sides of an element at once.</li>
+</ul>
```

Comment:
@pallavikadam-srijan Above text seems to be irrelevant to me
CC- @srijan-mehul-gada

Reply:
@surbhi90 could you please suggest any other relevant text which I can replace here or should I remove this text completely?